src/unit_test/nas_vxlan_unittest.cpp

```diff
@@ -0,0 +1,71 @@
+/*
```

Comment:
Want o add copyright?

Reply:
ignoring.. i got a script that will take care of copyright info, but will do it in next commit (rc0)